sdk/tests/conformance/extensions/oes-vertex-array-object.html

```diff
@@ -624,25 +624,31 @@
         gl.deleteBuffer(colorBuffer);
         gl.deleteBuffer(positionBuffer);
 
-        // The buffers should not be accessible at this point. Deleted objects that are bound
-        // in the current context undergo an automatic unbinding
+        var expectRetained = (ii != 0);
```

Comment:
Could you adjust the comment above rather than removing it? Maybe point out that the deleteBuffer calls are no-ops after the first iteration, so they are only automatically unbound from vaos[0], or something similar.